Update service-fabric-api-management-quick-start.md

damadei opened the pull request:
Removed the "Preview" for items which are not in preview anymore.
Added instructions to configure the vNET integration.

azurecla commented:
Hi __@stevestein__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-stste). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

damadei committed:
Update service-fabric-api-management-quick-start.md

Removed the "Preview" for items which are not in preview anymore.
Added instructions to configure the vNET integration.
